src/main/resources/init_db.sql

```diff
@@ -0,0 +1,8 @@
+-- CREATE TABLE books (
+-- 	id BIGINT AUTO_INCREMENT,
+--     title CHARACTER(50),
+--      price INT
+-- );
+
+-- USE book;
+--SELECT * FROM books;
```

Comment:
why commented?

Reply:
Because? when I was checking data in Workbench, I commented it, that it will be unworked